src/components/sbb-notification/sbb-notification.e2e.ts

```diff
@@ -73,22 +72,18 @@ describe('sbb-notification', () => {
     expect(element).to.be.null;
   });
 
-  // TODO: find out cause
-  it.skip('closes the notification and removes it from the DOM on close button click by keyboard', async () => {
+  it('closes the notification and removes it from the DOM on close button click by keyboard', async () => {
     const willCloseEventSpy = new EventSpy(SbbNotification.events.willClose);
     const didCloseEventSpy = new EventSpy(SbbNotification.events.didClose);
     const closeButton = element.shadowRoot.querySelector('.sbb-notification__close') as SbbButton;
 
     expect(element).not.to.be.null;
     expect(element).to.have.attribute('data-state', 'opened');
 
-    closeButton.focus();
-    await sendKeys({ down: 'Enter' });
-    await waitForLitRender(element);
+    closeButton.click();
```

Comment:
I think the previous test already does the same logic? but maybe this test does not give a lot of value anyways

Reply:
That was my mistake. Initially, it tested the "close by keyboard".
Anyway, I agree with you. The difference between keyboard and click interaction on an `sbb-button` is not the responsibility of the `sbb-notification`.
I think I'll remove it since it has a strange timing interaction problem that doesn't let the button be focused